app/code/Magento/LoginAsCustomerGraphQl/README.md

```diff
@@ -0,0 +1,51 @@
+# Module Magento LoginAsCustomerGraphQl
+
```

Comment:
Is all of the info in this README relevant for this module?

Reply:
@keharper  I have formatted readme file with the relevant content, Thanks